travis/install_google_cloud_sdk.sh

```diff
@@ -8,10 +8,12 @@ openssl aes-256-cbc -K $encrypted_aee7e38c959c_key -iv $encrypted_aee7e38c959c_i
     -in gcloud-client-secret.json.enc -out gcloud-client-secret.json -d
 
 # Download & install the Google Cloud SDK
-curl https://sdk.cloud.google.com | bash
+#curl https://sdk.cloud.google.com | bash
```

Comment:
Given that this happens in the before_install hook now, maybe call that out here and remove the commented out code here (and below)?